tests/NgTests/StorageFactoryTests.cs

```diff
@@ -56,8 +56,8 @@ public class StorageFactoryTests
                 { Arguments.CompressedStorageKeyValue, storageKeyValue },
                 { Arguments.CompressedStorageContainer, storageContainer },
                 { Arguments.CompressedStoragePath, storagePath },
-                { Arguments.StorageType, storageType},
-                { Arguments.UseCompressedStorage, "false"}
+                { Arguments.StorageType, storageType },
+                { Arguments.UseCompressedStorage, "false" }
```

Comment:
typos in method names in this class
`AzureCommpressedFactoryNull` -> `AzureCompressedFactoryNull`
`AzureCommpressedFactory` -> `AzureCompressedFactory`